Sources/ReadingListKit/ReadingListCell.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import DesignKit
+
+public struct ReadingListCell: View {
```

Comment:
would you think it's worth considering reusing existing `DiscoverKit` views, as reported in the requirements document?

Reply:
Yes, the current UI is placeholder and can be refined as we go.

Reply:
Since this UI is a placeholder, what do you think about tackling the reuse sooner rather than later? The reason for this would be to avoid duplicate work (1. building the ui in ReadingListKit, 2. "merge" it with the existing one)?

Reply:
imo we have bigger fish to fry right now, the placeholder UI can let us see if we are on the right path.
Once everything else is settled I would begin to prioritize polishing the UI.

If there is a concern around it looking bad we can just leave the ReadingList tab showing "Coming Soon" while we work on it. But I don't have that worry given the app is basically in Alpha I don't think it we can expect people to be too critical of incomplete work.

Reply:
I don't think the concern is how the views look, rather, if the plan is to reuse views (which is a requirement in the document), wouldn't you think that it's preferable to reuse them from the start, instead of building views that will then be replaced by existing ones?

Reply:
I feel like the rough draft let me get to the actual problem solving faster.

Reply:
Curious to know how building views and then discontinue them in favor of existing ones is faster than just using existing ones?